src/components/ChatLog.js

```diff
@@ -0,0 +1,22 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ChatLog = ({ entries, updateLikes }) => {
+  const ChatEntryComponents = entries.map((entry) => (
+    <ChatEntry
+      // key={entry.id}
+      sender={entry.sender}
+      body={entry.body}
+      timeStamp={entry.timeStamp}
+      updateLikes={updateLikes}/>
+  ));
```

Comment:
I'm curious as to why you commented out the key. Because `map()` creates sibling components, React prefers to have a key to uniquely identify each sibling component. For small data sets, using the id from the data is useful or even the timestamp. 

You pass all the tests, however, our linter gets cranky when we don't include the key and displays a `console.error` that isn't fatal, but adds extra noise to the results in our tests.